examples/customization/custom-configuration/haproxy/README.md

```diff
@@ -0,0 +1,27 @@
+# Customize the HAProxy configuration
+
+This example use a [ConfigMap](https://kubernetes.io/docs/user-guide/configmap/) to customize the HAProxy configuration.
+
+## Prerequisites
+
+This document has the following prerequisites:
+
+* Deploy a service named `ingress-default-backend` to be used as default backend service
+* Create a [TLS secret](/examples/PREREQUISITES.md#tls-certificates) named `tls-secret` to be used as default TLS certificate
```

Comment:
Link the deployment instructions instead the two items above:

---

* Deploy only the `tls-secret` and the default backend from the [deployment instructions](/examples/deployment/haproxy)